app/page.tsx

```diff
@@ -23,12 +23,12 @@ export default function Top({ searchParams }: { searchParams: { q: string } }) {
   const color = useColorModeValue('black', 'gray.300')
 
   const searchWord = searchParams.q
-  const uuid = useUserUuid()
+  const userId = useUserId()
 
   const { data, loading, fetchMore, refetch } = useTripsCollectionQuery({
     variables: {
       filter: {
-        uuid: { eq: uuid },
+        id: { eq: userId },
```

Comment:
We need to use `user_id` parameter to fetch trips.
Could you fix to use `user_id`?

```suggestion
        user_id: { eq: userId },
```

Reply:
ohh yes!! it's trip id... 
Thx !

Reply:
Fixed !